pkg/hintrunner/zero/zerohint.go

```diff
@@ -222,7 +222,12 @@ func GetHintFromCode(program *zero.ZeroProgram, rawHint zero.Hint, hintPC uint64
 		return createTestAssignHinter(resolver)
 	case findElementCode:
 		return createFindElementHinter(resolver)
+	case compareBytesInWordNondetCode:
+		return createCompareBytesInWordNondetHinter(resolver)
+	case compareKeccakFullRateInBytesNondetCode:
+		return createCompareKeccakFullRateInBytesNondetCode(resolver)
 	default:
+		fmt.Println(rawHint.Code)
```

Comment:
I think you can remove this line. But it could be a good idea to add the code of the unidentified hint in the error message. Could you pls do that?